### PR TITLE
Use aggregate initialisation for JetFinder

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@qgp @nzardosh this should actually work...

**Comment by qgp** [comment]
Duplicate of #4697

**Comment by qgp** [comment]
> fine with me, although this version handles the ghost* as before. i do not know if that is actually desirable, though.

@ktf What's wrong with default initialisation (instead of zero initialisation) for these variables?

**Comment by ktf** [comment]
Before your fix, the `ghost<something>` variables were not 0-initialised, but initialised starting from the respecive `<something>`. For example, `ghostEtaMin` was using `eta_Min` as initial value. I guess this was the reason / motivation behind the explicit template. You can still retain that behavior, but then you need to fix your aggregate initialisation using:

```C++
struct JetFinder {
   float etaMin = 0.9;
   ...
   float ghostEtaMin = etaMin;
   ...
};
```

This is what would have the same effect as the old constructor.

**Comment by ktf** [comment]
PR updated to show what I mean...

File: PWGJE/Core/JetFinder.h [hunk 1]
```diff
@@ -79,6 +79,7 @@ class JetFinder
   fastjet::RecombinationScheme recombScheme = fastjet::E_scheme;
   fastjet::Strategy strategy = fastjet::Best;
   fastjet::AreaType areaType = fastjet::active_area;
+
```
Comment (ktf) [inline PWGJE/Core/JetFinder.h:82]:
```suggestion
```

**Comment by qgp** [comment]
Sorry, then I missed your point before. Indeed, I discussed this with Nima and we deliberately chose to initialise the `ghost<sth>` with fixed values and not derive them from the constituent limits. I should have mentioned this in my PR, sorry for the omission.

**Comment by ktf** [comment]
Ok, fair enough. Thank you for understanding the original issue!

**Pull request closed without merging**